fluster/decoders/gstreamer.py

```diff
@@ -900,3 +900,15 @@ def __init__(self) -> None:
         self.api = "SW"
         self.caps = self.caps + ",format=S16LE"
         super().__init__()
+
+
+@register_decoder
+class FluendoLCEVCHWVAH264Decoder(GStreamer10Video):
```

Comment:
Let's keep the naming uniform, even though it has some redundancy. This will help us replace names easier in future refactoring
```suggestion
class FluendoFluLCEVCVAH264DecGst10Decoder(GStreamer10Video):
```